hasjob/views/helper.py

```diff
@@ -240,7 +240,7 @@ def record_views_and_events(response):
                     db.session.commit()
                 except IntegrityError:  # Race condition from parallel requests
                     db.session.rollback()
-            campaign_view_count_update.delay(campaign_id=campaign.id, user_id=g.user.id)
+            campaign_view_count_update.delay(datetime.utcnow(),campaign_id=campaign.id, user_id=g.user.id)
```

Comment:
Don't call `utcnow()` each time as it's a relatively slow call. Do `now = datetime.utcnow()` at the beginning of this function and pass that value each time.